thirdparty/mupdf/CMakeLists.txt

```diff
@@ -60,13 +57,19 @@ set(SHARED_BUILD_CMD sh -c "${SHARED_BUILD_STR}")
 set(PATCH_CMD1 sh -c "patch -N -p1 < ${CMAKE_CURRENT_SOURCE_DIR}/external_fonts.patch || true")
 # add support for encryted zips
 set(PATCH_CMD2 sh -c "patch -N -p1 < ${CMAKE_CURRENT_SOURCE_DIR}/encrypted_zip.patch || true")
+# use libjpeg as a shared library
+set(PATCH_CMD3 sh -c "patch -N -p1 < ${CMAKE_CURRENT_SOURCE_DIR}/libjpeg_shared.patch || true")
+# vec_dot fix (see https://github.com/ArtifexSoftware/mupdf/pull/5, needed for v1.12)
+set(PATCH_CMD4 sh -c "patch -N -p1 < ${CMAKE_CURRENT_SOURCE_DIR}/vec_dot.patch || true")
 
 # TODO: ignore shared git submodules built outside of mupdf by ourselves
+# https://git.ghostscript.com/mupdf.git is slow, so we use the official mirror on GitHub
+# v1.12 isn't tagged, so we refer to it with a hash
```

Comment:
The tag `1.12.0` seems to have been added two days ago, which I think is a lot more legible than that hash below. ;-)